src/app/public/modules/phone-field/types/number-return-format.ts

```diff
@@ -0,0 +1,4 @@
+/**
+ * Specifies the return format for the validated phone number. TODO: DENISE HELP WITH THE WORDING
```

Comment:
@Blackbaud-DeniseCisneros Thoughts?

Reply:
Is there a distinction between `return format` and just `format`?

Should we have JSDocs entries for the valid options in `SkyPhoneFieldNumberReturnFormat`? Seems like to me, but if not, we should probably explain the valid options here.